Tests/XCUITests/BookmarkingTests.swift

```diff
@@ -133,10 +133,10 @@ class BookmarkingTests: BaseTestCase {
         XCTExpectFailure("The app was not launched", strict: false) {
             waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
         }
-        typeOnSearchBar(text: "www.ebay")
+        typeOnSearchBar(text: "www.google")
         waitForExistence(app.tables["SiteTable"])
-        waitForExistence(app.tables["SiteTable"].cells.staticTexts["www.ebay"], timeout: 5)
-        XCTAssertTrue(app.tables["SiteTable"].cells.staticTexts["www.ebay"].exists)
+        waitForExistence(app.tables["SiteTable"].cells.staticTexts["www.google"], timeout: 5)
```

Comment:
I bet Google has better uptime than EBay. If Google is down, we have no work done anyway.